pyrefly/lib/test/lsp/semantic_tokens.rs

```diff
@@ -174,6 +174,47 @@ token-type: method
     );
 }
 
+#[test]
+fn deprecated_token_for_disabled_branch() {
```

Comment:
@asukaminato0721 I'm also having a hard time understanding what behavior is being tested here. Any chance you could help clarifying?

Reply:
It just ensures the tokens exist so editors can still style the dead code (the diagnostics system already marks the whole range as “unreachable”